javascript-functions/main.js

```diff
@@ -0,0 +1,43 @@
+function convertMinutesToSeconds(minutes, seconds) {
+  var multiply = minutes * seconds;
+  return multiply;
+}
+
+var convertMinutesToSecondsResult =
+convertMinutesToSeconds(5, 60);
+console.log('convertMinutesToSecondsResult:', convertMinutesToSecondsResult);
+
+function name(Hey, Beavis) {
+  var name = 'Hey, Beavis';
+  return name;
+}
+
+var nameResult = name('Beavis');
+console.log('nameResult:', nameResult);
+
+function getArea(width, height) {
+  var multiply = width * height;
+  return multiply;
+}
+
+var getAreaResult =
+getArea(17, 42);
+console.log('getAreaResult:', getAreaResult);
+
+function getFirstName(person) {
+  var getFirstName = 'Lelouche';
+  return getFirstName;
```

Comment:
It's still not returning `Ryan` if we invoke this function with `getFirstName({ firstName: 'Ryan', lastName: 'Choi' })` or `Katherine` with `getFirstName({ firstName: 'Katherine', lastName: 'Bustamante' })`